templates/infra/linodeCluster.yaml

```diff
@@ -15,3 +15,49 @@ spec:
     apiVersion: infrastructure.cluster.x-k8s.io/v1alpha2
     kind: LinodeFirewall
     name: ${CLUSTER_NAME}-nb
+  objectStore:
```

Comment:
I think we want a separate flavor for this rather than add it to the default flavor since this will increase the cost for creating a cluster